frontend/src/metabase/visualizations/lib/RowRenderer.js

```diff
@@ -117,7 +118,8 @@ export default function rowRenderer(
     .elasticX(true)
     .dimension(dimension)
     .group(group)
-    .ordering(d => d.index);
+    .ordering(d => d.index)
+    .othersLabel(t`Others`);
```

Comment:
We need an English native here. Pie chart is "Other", so we should probably just use that?

Reply:
hmmmm, I think "Others" works fine here. I think "Other" works for pie charts because you're looking at percentages.